### PR TITLE
Fix mousewheel

### PR DESCRIPTION
**Pull request by stephomi**
Tested on chrome, IE and firefox and it works (mousewheel)

### CODE REVIEW

**Comment by stephomi** [comment]
@jtorresfabra @Kuranes

File: sources/osgUtil/DisplayNodeGraphVisitor.js [hunk 1]
```diff
@@ -130,7 +130,11 @@ define( [
                 }
             }
 
+            if ( node.getSourceGeometry ) node.addChild( node.getSourceGeometry() );
+
             this.traverse( node );
+
```
Comment (cedricpinson) [inline sources/osgUtil/DisplayNodeGraphVisitor.js]: wtf ?

Comment (stephomi) [inline sources/osgUtil/DisplayNodeGraphVisitor.js]: It's temporary.
It's just one way to see if we have a morph or not when using the debug node graph tool.

Comment (stephomi) [inline sources/osgUtil/DisplayNodeGraphVisitor.js]: ok, don't merge I'll do something a bit better

**Pull request merged**